docs/references/api/deviceshifu-configmap.md

```diff
@@ -79,6 +82,21 @@ instructions: |
       ... # You can continue to configure commands and corresponding NodeID according to your own needs, just continue to add according to this format
 ```
 
+- **LwM2M**
+
+```yaml
+instructions: |
+  instructions:
+    float_value:      # The name of the command you want to set 
+      protocolPropertyList:
+        ObjectId: /3442/0/130
+        EnableObserve: false
```

Comment:
explain the Object Id and EnableObserve is, reference to https://github.com/Edgenesis/shifu.dev/blob/cd65935e2775555b67e70f623e83db7bd44752e2/docs/references/api/deviceshifu-configmap.md?plain=1#L64